src/avram/migrator/runner.cr

```diff
@@ -70,9 +70,24 @@ class Avram::Migrator::Runner
       raise <<-ERROR
       #{message}
 
-        #{green_arrow} If you are on macOS  you can install postgres tools from #{macos_postgres_tools_link}
-        #{green_arrow} If you are on linux you can try running #{linux_postgres_installation_instructions}
-        #{green_arrow} If you are on CI or some servers, there may already be a database created so you don't need this command"
+      Try this...
+
+        ▸ #{green_arrow} If you are on macOS  you can install postgres tools from #{macos_postgres_tools_link}
+        ▸ #{green_arrow} If you are on linux you can try running #{linux_postgres_installation_instructions}
+        ▸ #{green_arrow} If you are on CI or some servers, there may already be a database created so you don't need this command"
+
+      ERROR
+    elsif (message = e.message) && messages.includes?("could not connect to database template1")
+      raise <<-ERROR
+      Creating the database failed. It looks like Postgres is not running.
+
+      Message from Postgres:
+        #{message}
+
+      Try this...
+
+        ▸ Make sure Postgres is running
```

Comment:
Should we also mention the `verify_connection` task here?

Reply:
I don't think  so since this only requires a running db and `createdb`. Verify_connection wouldn't  ever work since the db  failed to be created